scripts/rssFeedGen.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+from email.Utils import formatdate
+import PyRSS2Gen
+import StringIO
+import re
+
+def extract_releases(html):
+    html_doc = open(html, 'r').read()
+    soup = BeautifulSoup(html_doc, "lxml")
+    divRelease = soup.find("div", {"id":"release-notes"})
+    releases = divRelease.findAll("div", {"id":re.compile("release-")})
+    return releases
+
+def id_to_pubDate(r):
+    id_str = r.findNext("span").get("id")
+    date_str = ''
+    for i in id_str.split():
+        for l in i:
+            if l.isdigit():
+                date_str = date_str + l
+    date_str = date_str + " 00:00:00"
+    date_obj = datetime.strptime(date_str, '%Y%m%d %H:%M:%S')
+    date_rss = datetime.strftime(date_obj, "%a, %d %b %Y %H:%M:%S %z")
+    return date_rss
+
+def extract_api_changes(r):
+    api_changes = r.findNext("a",{"href":re.compile("https://github.com/geoadmin/mf-chsdi3")}, text="Full changelog")
+    if api_changes != None:
+        api_change = "-Bug Fixes\n-Full changelog: " + api_changes.get("href")
+    else:
+        api_change = ''
+        for li in r.findAll("li"):
+            api_change = "- " + li.text + "\n  "
+    return api_change
+
+def extract_map_changes(r):
+    map_changes = r.findNext("a",{"href":re.compile("https://github.com/geoadmin/mf-geoadmin3")}, text="Full changelog")
+    if map_changes != None:
+        map_change = "-Bug Fixes\n-Full changelog: " + map_changes.get("href")
+    else:
+        map_change = ''
+        for li in r.findAll("li"):
+            map_change = "- " + li.text + "\n  "
+    return map_change
+
+def extract_geodata(r):
+    data = ''
+    table = r.find("tbody")
+    if table != None:
+        for row in table.findAll("tr"):
+            cols = row.findAll("td")
+            length = len(cols)
+            if length == 1:
+                data = cols[0].text
+            else:
+                status = cols[0].text
+                details = cols[1].text
+                data = data + status + ": " + details + "\n "
+    else:
+        data = "No change in the geodata"
+    return data
+      
+items = []
+
+releases = extract_releases("chsdi/static/doc/build/releasenotes/index.html")
+
+for r in releases:
+    # parse information from html
+    date_rss = id_to_pubDate(r)
+    api_change = extract_api_changes(r)
+    map_change = extract_map_changes(r)
+    data = extract_geodata(r)
+    # create feeds
+    items.append(PyRSS2Gen.RSSItem(
+            title = r.findNext("h2").text[:-1],
+            link = "//api3.geo.admin.ch/releasenotes/" + r.findNext("a").get("href"),
+            description = "API & APPLICATIONS\n API\n  Bugfixes\n  Full changelog: " + api_change + "\n MAP\n  Bugfixes\n  Full changelog: " + map_change + "\nGEODATA\n  " + data,
```

Comment:
See a RTS flux as an exemple:
https://www.rts.ch/espace-2/programmes/a-vous-de-jouer/?flux=rss

@davidoesch this is normal you can't read it as is.
@rfrancel If you want nicely formatted html in the description field you can use `prettify()` method of Beautifulsoup